Xamarin.Forms.Platform.Android/BackgroundManager.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.ComponentModel;
+using Xamarin.Forms.Internals;
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	internal static class BackgroundManager
+	{
+		public static void Init(IVisualElementRenderer renderer)
+		{
+			_ = renderer ?? throw new ArgumentNullException($"{nameof(BackgroundManager)}.{nameof(Init)} {nameof(renderer)} cannot be null");
+
+			renderer.ElementPropertyChanged += OnElementPropertyChanged;
+			renderer.ElementChanged += OnElementChanged;
+		}
+
+		public static void Dispose(IVisualElementRenderer renderer)
+		{
+			_ = renderer ?? throw new ArgumentNullException($"{nameof(BackgroundManager)}.{nameof(Init)} {nameof(renderer)} cannot be null");
+
+			renderer.ElementPropertyChanged -= OnElementPropertyChanged;
+			renderer.ElementChanged -= OnElementChanged;
+		}
+
+		static void UpdateBackgroundColor(AView Control, VisualElement Element, Color? color = null)
+		{
+			if (Element == null || Control == null)
+				return;
+
+			Control.SetBackgroundColor((color ?? Element.BackgroundColor).ToAndroid());
+		}
+
+
+		static void OnElementChanged(object sender, VisualElementChangedEventArgs e)
+		{
+			var reference = Guid.NewGuid().ToString();
```

Comment:
Nope, we don't do this anymore. `Performance.Start` has `reference` as an `out` now.